en/components/general/getting_started.md

```diff
@@ -78,7 +88,7 @@ ig
     <p style="text-align:center;">Building Your First Ignite UI CLI App</p>
 </div>
 
-Alternatively you can use the Ignite UI CLI [`commands`](cli-overview.md#commands) for generating an Ignite UI project, adding a new component or building and serving the application.
+Alternatively you can use the Ignite UI Schematics collection or CLI commands for generating an Ignite UI project, adding a new component or building and serving the application.
```

Comment:
Eh same comment - Angular Schematics (general) or Ignite UI for Angular Schematic (the ones for this product). And CLI applies for both and the building part is iffy. Again this should go, whatever it's trying to explain isn't helping.
At best you can turn this into:
```suggestion
Learn more about our [Angular Schematics & Ignite UI CLI](cli-overview.md).
```
Or it should go too.